core/egov-localization/src/main/java/org/egov/domain/service/MessageService.java

```diff
@@ -104,13 +107,50 @@ public void bustCache() {
         return messages;
     }
     
-    public boolean createMessage(CreateMessagesRequest createMessagesRequest){
-    	messageRepository.deleteMessages(createMessagesRequest);
-   		boolean createStatus = messageRepository.createMessage(createMessagesRequest);
+    public boolean createMessage(NewMessagesRequest newMessagesRequest){
```

Comment:
@darshannagesh The domain layer should not be aware of the web layer. In the controller convert the createMessagesRequest to a list of domain Message list and then invoke the service.